--- a/drivers/gpu/drm/Kconfig	2025-07-28 00:26:38.000000000 +0300
+++ b/drivers/gpu/drm/Kconfig	2025-07-30 18:02:18.864934909 +0300
@@ -267,20 +267,8 @@
 
 source "drivers/gpu/drm/sysfb/Kconfig"
 
-source "drivers/gpu/drm/arm/Kconfig"
-
-source "drivers/gpu/drm/radeon/Kconfig"
-
 source "drivers/gpu/drm/amd/amdgpu/Kconfig"
 
-source "drivers/gpu/drm/nouveau/Kconfig"
-
-source "drivers/gpu/drm/nova/Kconfig"
-
-source "drivers/gpu/drm/i915/Kconfig"
-
-source "drivers/gpu/drm/xe/Kconfig"
-
 source "drivers/gpu/drm/kmb/Kconfig"
 
 config DRM_VGEM