--- a/drivers/gpu/drm/Kconfig    2025-04-28 02:17:36.835612668 +0300
+++ b/drivers/gpu/drm/Kconfig    2025-04-28 02:18:29.968610257 +0300
@@ -328,18 +328,8 @@
 
 source "drivers/gpu/drm/i2c/Kconfig"
 
-source "drivers/gpu/drm/arm/Kconfig"
-
-source "drivers/gpu/drm/radeon/Kconfig"
-
 source "drivers/gpu/drm/amd/amdgpu/Kconfig"
 
-source "drivers/gpu/drm/nouveau/Kconfig"
-
-source "drivers/gpu/drm/i915/Kconfig"
-
-source "drivers/gpu/drm/xe/Kconfig"
-
 source "drivers/gpu/drm/kmb/Kconfig"
 
 config DRM_VGEM
