--- a/drivers/hwmon/Kconfig	2025-04-28 01:55:17.897953012 +0300
+++ b/drivers/hwmon/Kconfig	2025-04-28 01:55:19.719748159 +0300
@@ -868,9 +868,10 @@
 
 config SENSORS_IT87
 	tristate "ITE IT87xx and compatibles"
-	depends on HAS_IOPORT
+	depends on HAS_IOPORT && BROKEN
 	depends on !PPC
 	select HWMON_VID
+        default n
 	help
 	 If you say yes here you get support for ITE IT8705F, IT8712F, IT8716F,
 	 IT8718F, IT8720F, IT8721F, IT8726F, IT8728F, IT8732F, IT8758E,
