From 8a6afc4ac94ac50fbcf3a6d2295e084f97e1a5a9 Mon Sep 17 00:00:00 2001
From: SteavenGamerYT <62666559+SteavenGamerYT@users.noreply.github.com>
Date: Wed, 30 Jul 2025 20:16:31 +0300
Subject: [PATCH] Update zenergy.c

---
 zenergy.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/zenergy.c b/zenergy.c
index 0a66df6..f6dd4d1 100644
--- a/zenergy.c
+++ b/zenergy.c
@@ -80,7 +80,7 @@ static void get_energy_units(struct zenergy_data *data)
 {
 	u64 rapl_units;
 
-	rdmsrl_safe(ENERGY_PWR_UNIT_MSR, &rapl_units);
+	 rdmsrq_safe(ENERGY_PWR_UNIT_MSR, &rapl_units);
 	data->energy_units = (rapl_units & zenergy_UNIT_MASK) >> 8;
 }
 
@@ -89,7 +89,7 @@ static void __accumulate_delta(struct sensor_accumulator *accum,
 {
 	u64 input;
 
-	rdmsrl_safe_on_cpu(cpu, reg, &input);
+	rdmsrq_safe_on_cpu(cpu, reg, &input);
 	input &= zenergy_MASK;
 
 	if (input >= accum->prev_value)
